setup.py

```diff
@@ -1,14 +1,14 @@
 from setuptools import setup
 
 setup(
-    name="data-structures",
-    description="this module will be expanded as we write more data-structures",
-    version=1.1,
-    author="Ted Callahan and Mark Kessler-Wenicker",
+    name="Codefellows Seattle Advanced Python 401 Course - Data Structure Assignments",
```

Comment:
This is not a good package name.  Keep the name short and descriptive.  Add additional info to the description.